clients/python/test/e2e/ci/e2e/e2e/postprocess.py

```diff
@@ -111,20 +111,17 @@ def single_testrun(exit_code: int) -> None:
 
 @cli.command()
 @handle_validation_error
-def check_for_failure():
+def check_for_failure(e2e_artifacts_dir: str):
     """Loop through all json artifacts and fail in case of testfailure"""
-    pytest_ini: PytestIniFile = PytestIniFile.create_from_file()
-    artifacts: Artifacts = pytest_ini.artifacts
-    if not artifacts.log_dir.is_dir():
+    artifacts: Path = Path(e2e_artifacts_dir)
+    if not artifacts.is_dir():
         raise typer.Exit(code=E2eExitCodes.CI_SCRIPT_FAILURE)
-    result_jsons = list(Path(artifacts.artifact_dir).glob("*.json"))
+    result_jsons = list(artifacts.glob("*.json"))
```

Comment:
TIP: `glob` returns a generator and it is better to iterate it the loop instead of just create a list of it.  Note that the list of objects found is **unbound** and you and me know first-hand that: that is very bad :-)
A good practice would be

```python

count = 0
for path in artifacts.glob("*.json"):
     count+=1
     # do something here ...
     
if not count:
     raise typer.Exit(code=E2eExitCodes.CI_SCRIPT_FAILURE)
```

Reply:
Good point. I will fix.
Thanks